README.md

```diff
@@ -25,6 +25,10 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Lint OpenAPI file
-        uses: ./visku-api-validator
+        uses: Bis-Henderson/visku-api-validator/@v1
```

Comment:
why not Visku?

Reply:
because our github org name is Bis-Henderson, not visku

Reply:
Damn good point